flume-ng-channels/flume-file-channel/src/main/java/org/apache/flume/channel/file/instrumentation/FileChannelCounterMBean.java

```diff
@@ -23,4 +23,14 @@
 public interface FileChannelCounterMBean extends ChannelCounterMBean {
 
   boolean isOpen();
+
+  int getClosed();
+
+  int getUnhealthy();
+
+  long getEventPutErrorCount();
```

Comment:
Please add Javadoc for each metric in this interface. It should describe the purpose and behavior of the metric.

For example:

```
/** A count of the number of errors encountered while trying to put() onto the channel,
  * including IOExceptions and corruption-related errors.
  */
long getEventPutErrorCount();
```